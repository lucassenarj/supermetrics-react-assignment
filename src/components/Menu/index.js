import React from "react";
import MenuItem from "./../MenuItem";
import "./index.scss";

function Menu() {
  return (
    <nav className="menu">
      <MenuItem path="/dashboard" text="Blog" />
      <MenuItem path="/about" text="About" />
      <MenuItem path="/contact" text="Contact" />
    </nav>
  );
};

export default Menu;
