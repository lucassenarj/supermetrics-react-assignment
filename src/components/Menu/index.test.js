import React from "react";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router";
import Menu from "./";

describe("Components", () => {
  describe("Menu", () => {
    it("renders correctly", () => {
      const { container } = render(
        <MemoryRouter>
          <Menu />
        </MemoryRouter>
      );

      expect(container.firstChild).toMatchSnapshot();
    });
  });
});
