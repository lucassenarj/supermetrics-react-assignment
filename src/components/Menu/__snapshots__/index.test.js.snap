// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Menu renders correctly 1`] = `
<nav
  class="menu"
>
  <li
    class="menu-item"
  >
    <a
      href="/dashboard"
    >
      Blog
    </a>
  </li>
  <li
    class="menu-item"
  >
    <a
      href="/about"
    >
      About
    </a>
  </li>
  <li
    class="menu-item"
  >
    <a
      href="/contact"
    >
      Contact
    </a>
  </li>
</nav>
`;
